sdk/resourcemanager/azure-resourcemanager-appservice/src/main/java/com/azure/resourcemanager/appservice/implementation/FunctionAppsImpl.java

```diff
@@ -62,13 +55,20 @@ public Mono<FunctionApp> getByResourceGroupAsync(final String resourceGroupName,
         return this
             .getInnerAsync(resourceGroupName, name)
             .flatMap(
-                siteInner ->
-                    Mono
-                        .zip(
-                            this.inner().getConfigurationAsync(resourceGroupName, name),
-                            this.inner().getDiagnosticLogsConfigurationAsync(resourceGroupName, name),
-                            (SiteConfigResourceInner siteConfigResourceInner, SiteLogsConfigInner logsConfigInner) ->
-                                wrapModel(siteInner, siteConfigResourceInner, logsConfigInner)));
+                siteInner -> this.inner().getConfigurationAsync(resourceGroupName, name)
+                    .flatMap(
+                        siteConfigResourceInner -> {
+                            if (FunctionAppImpl.isFunctionAppOnACA(siteInner)) {
+                                return Mono.just(wrapModel(siteInner, siteConfigResourceInner, null));
+                            } else {
+                                return this.inner().getDiagnosticLogsConfigurationAsync(resourceGroupName, name)
+                                    .flatMap(
+                                        logsConfigInner ->
+                                            Mono.just(wrapModel(siteInner, siteConfigResourceInner, logsConfigInner))
+                                    );
```

Comment:
`.flatMap(.. Mono.just())` is it same as `.map`?

Reply:
Yeah. Changed to `map`.